Plugins/Flow.Launcher.Plugin.BrowserBookmark/Main.cs

```diff
@@ -88,35 +107,45 @@ public List<Result> Query(Query query)
 
         private static Channel<byte> refreshQueue = Channel.CreateBounded<byte>(1);
 
-        private async Task MonitorRefreshQueue()
+        private static SemaphoreSlim fileMonitorSemaphore = new(1, 1);
+
+        private static async Task MonitorRefreshQueueAsync()
         {
+            if (fileMonitorSemaphore.CurrentCount < 1)
+            {
+                return;
+            }
+            await fileMonitorSemaphore.WaitAsync();
             var reader = refreshQueue.Reader;
             while (await reader.WaitToReadAsync())
             {
-                await Task.Delay(2000);
+                await Task.Delay(5000);
```

Comment:
Do we need this delay? I've tested without it and had no issues...

Reply:
> Do we need this delay? I've tested without it and had no issues...

IDK to be honest. May be for cases like modifiying mutiple bookmarks in a short time?

Reply:
That should be taken care of by the bounded channel we use as a queue.

@taooceros do we remove the delay?

Reply:
I think it is for multiple event triggering. When I test it, it seems like there will be multiple event being triggered in some cases. I am not sure about the execution order, so maybe the channel bound won't help. However, it shouldn't be a large issue. I think the delay can be very short or even removed. It's not gonna make a big deal (even reindex twice is fast).

Reply:
@VictoriousRaptor maybe a very short delay (500ms) will suffice. Have you experienced that this method got called twice?

Reply:
> @VictoriousRaptor maybe a very short delay (500ms) will suffice. Have you experienced that this method got called twice?

I can't remember my exact test result but after removing `LastAccessed` notify filter in  d0289b49b669140b4e365817b0a452fe2f7e98ad it's no longer called twice. Sorry but VS is broken somehow and I can't test it now.

Reply:
Ah sounds good. Then let's remove it.